app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/confirmation_of_draft_return.scala.html

```diff
@@ -0,0 +1,50 @@
+@*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.cgtpropertydisposalsfrontend.config.ViewConfig
+@import uk.gov.hmrc.cgtpropertydisposalsfrontend.controllers.actions.RequestWithSessionData
+@import uk.gov.hmrc.cgtpropertydisposalsfrontend.controllers.accounts
+@import uk.gov.hmrc.cgtpropertydisposalsfrontend.controllers.returns
+@import uk.gov.hmrc.cgtpropertydisposalsfrontend.models.returns.DraftReturn
+@import uk.gov.hmrc.cgtpropertydisposalsfrontend.models.LocalDateUtils
+@this(
+mainTemplate: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.main_template,
+pageHeading: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.page_heading,
+backLinkComponent: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.back_link
+)
+
+@(draftReturn: DraftReturn)(implicit request: RequestWithSessionData[_], messages:Messages, viewConfig: ViewConfig)
+@title = @{messages("confirmDraftReturn.title")}
+@message = @{messages("confirmDraftReturn.message", LocalDateUtils.govDisplayFormat(draftReturn.lastUpdatedDate.plusDays(29)))}
+
+@mainTemplate(title = title, userType = request.userType) {
+@backLinkComponent(returns.routes.TaskListController.taskList())
+
+@pageHeading(title)
+<div class="notice" style="margin: 30px 0;">
```

Comment:
@nubz  could you double check this template please? You'd know better than I would!